upgrades/v1.4.2.md

```diff
@@ -19,7 +19,6 @@ First of all, pull the latest kroma-up and checkout to v1.4.2.
 
 ```bash
 git pull origin main
```

Comment:
```suggestion
git fetch origin
git checkout v1.4.2
```

Reply:
Both should be done.
```suggestion
git fetch origin
git pull origin main
```

Reply:
Yes the `git fetch origin` solve it
![fetch](https://github.com/kroma-network/kroma-up/assets/119992979/33897ab5-a815-46cc-8d90-00fedcf2ad07)